src/app/components/CopyToClipboard/index.tsx

```diff
@@ -44,12 +84,8 @@ export const CopyToClipboard: FC<CopyToClipboardProps> = ({ label, value }) => {
       onClick={handleCopyToClipboard}
       sx={{ display: 'inline-flex', alignItems: 'center' }}
     >
-      {label || value}
-      <Tooltip arrow onOpen={hideTooltip} open={isCopied} placement="top" title={t('clipboard.success')}>
-        <IconButton color="inherit" aria-label={ariaLabel}>
-          <ContentCopyIcon fontSize="small" sx={{ color: COLORS.brandDark }} />
-        </IconButton>
-      </Tooltip>
+      {label || (isMobile ? trimLongString(value) : value)}
+      <CopyToClipboardButton value={value} />
```

Comment:
There is one click handler in parent Box and now another one within CopyToClipboardButton. Due to event capturing this will trigger both handlers (two times copy two clipboard etc). Additionally, there is no use of `timeout` in `CopyToClipboard` currently.